Src/FluentAssertions.nuspec

```diff
@@ -22,41 +22,20 @@
       See https://github.com/dennisdoomen/fluentassertions/releases/
     </releaseNotes>
     <frameworkAssemblies>
-      <frameworkAssembly assemblyName="System.Xml" targetFramework="portable-net45+monoandroid+xamarin.ios10+monotouch" />
-      <frameworkAssembly assemblyName="System.Xml.Linq" targetFramework="portable-net45+monoandroid+xamarin.ios10+monotouch" />
+      <frameworkAssembly assemblyName="System.Xml" targetFramework="net45" />
+      <frameworkAssembly assemblyName="System.Xml.Linq" targetFramework="net45" />
     </frameworkAssemblies>
     <dependencies>
       <group targetFramework="net4" />
-      <group targetFramework="sl5" />
-      <group targetFramework="win8" />
-      <group targetFramework="wp8" />
-      <group targetFramework="wpa81" />
-      <group targetFramework="xamarin.ios" />
-      <group targetFramework="monotouch" />
-      <group targetFramework="monoandroid" />
       <group targetFramework="portable-net40+sl5+win8+wp8+wpa81" />
-      <group targetFramework="dotnet">
```

Comment:
Looks that way. That tool can be removed from the build process as it's no longer needed.